src/components/ButtonPanel.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable arrow-body-style */
```

Comment:
Is there a reason why you disabled this? :thinking: 

I think removing the curly brackets and the `return` keyword should fix this.